Essentials/src/messages.properties

```diff
@@ -427,6 +427,7 @@ powerToolRemove=\u00a76Command \u00a7c{0}\u00a76 removed from \u00a7c{1}\u00a76.
 powerToolRemoveAll=\u00a76All commands removed from \u00a7c{0}\u00a76.
 powerToolsDisabled=\u00a76All of your power tools have been disabled.
 powerToolsEnabled=\u00a76All of your power tools have been enabled.
+priceChanged=\u00a76Price changed from {0} to {1}.
```

Comment:
I feel like this message could be a little more useful/explanatory

Reply:
I could add the entity name